frontend/src/components/NavBarDropdown.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+import React, { useState } from "react";
+import { signOut } from "next-auth/react";
+import { LogOut, User } from "react-feather";
+
+export default function Dropdown(props: { initials: string }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  function toggle() {
+    setIsOpen((old) => !old);
+  }
+
+  const transClass = isOpen ? "flex" : "hidden";
+
+  return (
+    <>
+      <div className="relative">
+        <button
+          className="flex rounded-full border border-white h-9 w-9 overflow-hidden justify-center items-center"
+          onClick={toggle}
+        >
+          <p className="text-white body-small">{props.initials}</p>
+        </button>
+        <div
+          className={`absolute origin-right right-0 top-11 rounded text-primary_default bg-white flex flex-col w-[138px] shadow-[0_4px_4px_0_rgba(66,61,137,0.10)]  ${transClass}`}
```

Comment:
Hva gjør origin-right her? 😅

Reply:
Her tror jeg også det blir mer riktig å bruke rounded-b, siden du kun ønsker rounded i bunn av div 👍

Reply:
Men vil ikke right-0 sørge for at høyre kant på dropdown settes på høyre kant på parent div? 😅

Reply:
Fikset i parprogrammering